src/main/config.py

```diff
@@ -76,23 +77,21 @@
 # Utilities - DO NOT TOUCH:
 mac = 'Darwin'  # Mac OS
 windows = 'Windows'  # Windows
-slash = ''  # Leave empty
-save_path = None  # Leave empty
+save_path = ''  # Leave empty
 
 # Save_path utilities and os compatibility
 if platform.system() == mac:
     save_path = save_path_mac
-    slash = '/'
+
 elif platform.system() == windows:
     save_path = save_path_windows
-    slash = '\\'
 
 # Paths to folders
-batch_save_path = save_path + slash + 'Batch_Json_files'  # The output path for batches generated by Batch_Sorter.py
-images_save_path = save_path + slash + 'NFT_Image_Output'  # The output path for images generated by Image_Generator.py
-modelAssetPath = save_path + slash + "3D_Model_Input"  # The input path for 3D models
-model_save_path = save_path + slash + "3D_Model_Output"  # The output path for 3D models generated by Model_Generator.py
-model_Script_Ignore_Path = modelAssetPath + slash + "Script_Ignore_Folder"  # The path to the Script_Ignore_Folder for 3D models
+batch_save_path = os.path.join(save_path, 'Batch_Json_files')  # The output path for batches generated by Batch_Sorter.py
+images_save_path = os.path.join(save_path, 'NFT_Image_Output')  # The output path for images generated by Image_Generator.py
+modelAssetPath = os.path.join(save_path, '3D_Model_Input')  # The input path for 3D models
+model_save_path = os.path.join(save_path, '3D_Model_Output')  # The output path for 3D models generated by Model_Generator.py
+model_Script_Ignore_Path = os.path.join(modelAssetPath, 'Script_Ignore_Folder')  # The path to the Script_Ignore_Folder for 3D models
```

Comment:
Did this work? I had assumed not since blender needs two slashes like `test\\path`

Reply:
I believe this will work, however there are other instances in other folders that need to be changes to this format.

Reply:
@FastestMolasses Could you go through all the other files and reformat all instances of "slash" to the above method? Removing it as you did here without implementing os.path.join() in the other instances will break a lot of systems.